config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources :users do
+    resources :posts
+  end
 end
```

Comment:
- [ ] Kindly configure your routes properly, so you don't expose unnecessary endpoints

```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```